### ISSUE TITLE
Normalization issue of words containing œ (in French)

### ISSUE THREAD

**Issue by scossin** [scossin/IAMsystem#11]
```java
TermDetector detector = new TermDetector();
Term term = new Term("oedeme","I50");
detector.addTerm(term);
System.out.println(detector.detect("œdeme").getCTcodes().size());
```
This is not a normalization issue since normalization requires a one-2-one mapping: character -> normalized character. 
A way to handle it is with abbreviations: 

```java
TermDetector detector = new TermDetector();
Term term = new Term("oedeme","I50");
Abbreviations abb = new Abbreviations();
abb.addAbbreviation("oedeme", "œdeme");
detector.addTerm(term);
detector.addFuzzyAlgorithm(abb);
System.out.println(detector.detect("œdeme").getCTcodes().size());
```

A general fix could be to add a method in the Abbreviations class to add automatically words synonyms: it takes in input the unique tokens of the terminology, if a token contains 'œ' then create a long form by replacing 'œ' by oe. 
This method will also handle similar cases.

**Comment by scossin** [comment]
It makes more sense to move first the method *getUniqueToken* (defined in StringEncoderSyn and PrefixTrie) to the Terminology class then call this method from the abbreviation class

**Comment by scossin** [comment]
fix in version 2.1.0
